### ISSUE TITLE
Add pyramid list and detail views

### ISSUE THREAD

**Issue by MyleneSimon** [usnistgov/WIPP-frontend#13]
**Description**
Add the "Pyramid" data type to the WIPP-frontend, with list page and detail page.

**Proposal**
List view, detail view (similar to what was in WIPP 2.xx, except for the actual deep zoom view that will be treated in a separate issue)

**Additional context**
Will have to wait for [the backend integration of pyramids](https://github.com/usnistgov/WIPP-backend/issues/4) to be done before starting working on this issue.
